From 6b2ccab74292935f7687bc5f23ab6775fbf5f960 Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Wed, 17 Aug 2022 21:41:52 -0700
Subject: [PATCH] depends: Remove git dependency

Git command fails if the source directory is not a git repo but is
a subdirectory of one.
---
 tools/depends/xbmc-addons.include | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/depends/xbmc-addons.include b/tools/depends/xbmc-addons.include
index a3c23a9761..f2d766c89a 100644
--- a/tools/depends/xbmc-addons.include
+++ b/tools/depends/xbmc-addons.include
@@ -71,7 +71,7 @@ distclean:
 	rm -rf $(PLATFORM) .installed-$(PLATFORM) native
 
 .installed-$(PLATFORM): $(DEPS)
-	cd $(ADDON_PROJECT_DIR) && (git clean -xfd || rm -rf CMakeCache.txt CMakeFiles cmake_install.cmake build/*)
+	cd $(ADDON_PROJECT_DIR) && rm -rf CMakeCache.txt CMakeFiles cmake_install.cmake build/*
 	mkdir -p $(PLATFORM)
 ifeq ($(PREFIX),)
 	@echo
-- 
2.34.1

